### PR TITLE
Release notes for 1.1

### PR DESCRIPTION
**Pull request by hovsepm**
## Description

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ ] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md).**
- [ ] **The pull request does not introduce [breaking changes](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/Documentation/breaking-changes.md).**

### [General Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#general-guidelines)
- [ ] Title of the pull request is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#testing-guidelines)
- [ ] Pull request includes test coverage for the included changes.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#sdk-generation-guidelines)
- [ ] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [ ] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as the commitid of your swagger spec or link to the swagger spec, used to generate the code.
- [ ] The `*.csproj` and `AssemblyInfo.cs` files have been updated with the new version of the SDK.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,8 +17,8 @@ The Azure Management Libraries for .NET is a higher-level, object-oriented API f
   <tr>
     <td>Compute</td>
     <td>Virtual machines and VM extensions<br>Virtual machine scale sets<br>Managed disks</td>
-    <td></td>
     <td valign="top">Azure container services<br>Azure container registry</td>
```
Comment (milismsft) [inline README.md]: "Azure container registry" has its own package and it's not part of compute. Should be either in its own raw or moved under "More services"

File: README.md [hunk 2]
```diff
@@ -248,6 +263,25 @@ You can find plenty of sample code that illustrates management scenarios (69+ en
...
+<li><a href="https://github.com/Azure-Samples/aad-dotnet-manage-service-principals">Manage service principals using Java</a></li>
+<li><a href="https://github.com/Azure-Samples/aad-dotnet-browse-graph-and-manage-roles">Browse graph (users, groups and members) and managing roles</a></li>
+<li><a href="https://github.com/Azure-Samples/aad-dotnet-manage-passwords">Manage passwords</li>
+</ul></td>
+  </tr>
+<tr>
+    <td>Container Service and Container Registry</td>
+    <td><ul style="list-style-type:circle">
+<li><a href="https://github.com/Azure-Samples/acr-dotnet-manage-azure-container-registry">Manage container registry</a></li>
+<li><a href="https://github.com/Azure-Samples/acs-dotnet-deploy-image-from-acr-to-kubernetes">Deploy an image from container registry to Kubernetes cluster</a></li>
```
Comment (milismsft) [inline README.md]: Samples not available at this time; lines 278 through 281 (inclusive) should be removed/commented out using <! thing

File: README.md [hunk 3]
```diff
@@ -327,6 +361,7 @@ You can find plenty of sample code that illustrates management scenarios (69+ en
     <td>App Service - Web Apps on <b>Linux</b></td>
     <td><ul style="list-style-type:circle">
 <li><a href="https://github.com/Azure-Samples/app-service-dotnet-manage-web-apps-on-linux">Manage Web apps</a></li>
+<li><a href="https://github.com/Azure-Samples/app-service-dotnet-deploy-image-from-acr-to-linux">Deploy a container image from Azure Container Registry to Linux containers</a></li>
```
Comment (milismsft) [inline README.md]: Sample not available at this time; line 364 should be remove/commented out via <!

File: README.md [hunk 4]
```diff
@@ -30,39 +30,54 @@ The Azure Management Libraries for .NET is a higher-level, object-oriented API f
...
+    .WithRegion(Region.USEast)
+    .WithNewResourceGroup(rgName)
+    .WithKind(DatabaseAccountKind.GlobalDocumentDB)
+    .WithSessionConsistency()
+    .WithWriteReplication(Region.USWest)
+    .WithReadReplication(Region.USCentral)
+    .Create();
+```
+
+#### Create a Virtual Machine
```
Comment (milismsft) [inline README.md]: @selvasingh Should we also add a code snippet for creating an Azure Container Registry and Azure Container Service (with Kubernetes)?

File: README.md (hunk 1 above)
Comment (hovsepm) [inline README.md]: Fixed. Kind of :)

File: README.md (hunk 3 above)
Comment (hovsepm) [inline README.md]: Commented.

File: README.md (hunk 2 above)
Comment (hovsepm) [inline README.md]: Fixed

File: README.md (hunk 4 above)
Comment (hovsepm) [inline README.md]: rejected

**Pull request merged**